Makefile

```diff
@@ -48,23 +48,11 @@ vendor:
 	patch -p1 < 0002-Sort-machines-before-syncing.patch
 
 .PHONY: generate
-generate: gendeepcopy generate-mocks
+generate: 
+	go generate ./pkg/... ./cmd/...
```

Comment:
So `go generate` now can run the `deepcopy-gen` binary? Resp. why was `gendeepcopy` removed from the Makefile?

Reply:
See https://github.com/openshift/cluster-api-provider-aws/blob/master/pkg/apis/apis.go#L18
and deepcopy-gen is required by Gopkg. 
This is all driven by kubebuilder generation